sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -231,7 +231,7 @@ protected static String getQuotedSubscriptLiteral(String nodeName) {
         }
     }
 
-    private class InnerExpressionAnalyzer extends AstVisitor<Symbol, ExpressionAnalysisContext> {
+    protected class InnerExpressionAnalyzer extends AstVisitor<Symbol, ExpressionAnalysisContext> {
```

Comment:
Because otherwise I cannot use it here:
https://github.com/crate/crate/pull/4421/files/def177ff6d4857d1a9c2d1d9c37859717bb5c57f#diff-bf12c5e61f9b0fe950aa518b1ed601abR566